lib/tasks/users_to_ldap.rake

```diff
@@ -4,7 +4,7 @@ namespace :users_to_ldap do
     base = 'cn=users,cn=accounts,dc=bedlamtheatre,dc=co,dc=uk'
 
 
-    auth = { method: :simple, username: Rails.application.secrets.ldap[:bind_user], password: Rails.application.secrets.ldap[:bind_pass] }
+    auth = { method: :simple, username: Rails.application.credentials[:ldap][Rails.env.to_sym][:bind_user], password: Rails.application.credentials[:ldap][Rails.env.to_sym][:bind_pass] }
```

Comment:
do we still need the LDAP module btw?

Reply:
this feels like an artifact from a removed feature. guess it's not doing any harm

Reply:
It does seem to be a very old rake task, but because it's just a task, I don't really mind